webapp/src/components/Solution.vue

```diff
@@ -5,14 +5,14 @@
 </template>
 
 <script>
-import Solution from "@/services/solutions.service";
+import solution from "@/services/solutions.service";
```

Comment:
Not so nice. Acho `solutionService` ou `service` ajuda a distinguir o serviço do componente `Solution` em si.

Reply:
E também acho que nossos imports deveriam ter a primeira letra maíuscula. Na minha opinião deveria ser `SolutionService`.

Reply:
ta feito o carreto

Reply:
7587c552ae5d6e782c8af7f88c56aa04ebfad1ce